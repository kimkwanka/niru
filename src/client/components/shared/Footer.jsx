import React from 'react';

const year = new Date().getFullYear();

const Footer = () => (
  <footer>
    <div className="container flex-column items-center">
      <h5 className="margin-top-small">This is a sticky footer!</h5>
      <h5>
        <a
          href="https://github.com/kimkwanka/niru"
          target="_blank" rel="noopener noreferrer"
        >
        niru
        </a> by Kim Kwanka &copy; {year}
      </h5>
    </div>
  </footer>
);

export default Footer;
