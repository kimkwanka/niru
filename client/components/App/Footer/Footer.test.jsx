import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Footer from './Footer';

describe('<Footer /> - Snapshot', () => {
  it('matches its snapshot', () => {
    const wrapper = shallow(<Footer />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
