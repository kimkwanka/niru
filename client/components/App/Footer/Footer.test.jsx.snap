// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> - Snapshot matches its snapshot 1`] = `
<footer>
  <div
    className="container flex-column items-center"
  >
    <h5
      className="margin-top-small"
    >
      This is a sticky footer!
    </h5>
    <h5>
      <a
        href="https://github.com/kimkwanka/niru"
        rel="noopener noreferrer"
        target="_blank"
      >
        niru
      </a>
       by Kim Kwanka Â© 2019
    </h5>
  </div>
</footer>
`;
